rpi_backlight.py

```diff
@@ -45,38 +47,53 @@ def _set_value(name: str, value: Any) -> None:
             _perm_denied()
 
 
+def __set_brightness_value(value: int) -> None:
```

Comment:
One leading underscore is the convention, read more about one vs two leading underscores here: https://dbader.org/blog/meaning-of-underscores-in-python

Reply:
Also, remember to change all `set_brightness_value` calls to `_set_brightness_value`!

Reply:
fixed

Reply:
Thanks for teaching!